scipy/_build_utils/compiler_helper.py

```diff
@@ -64,3 +66,40 @@ def try_add_flag(args, compiler, flag, ext=None):
     """Appends flag to the list of arguments if supported by the compiler"""
     if try_compile(compiler, flags=args+[flag], ext=ext):
         args.append(flag)
+
+def set_cxx_flags_hook(build_ext, ext):
+    """Sets basic compiler flags for compiling C++11 code"""
+    cc = build_ext._cxx_compiler
+    args = ext.extra_compile_args
+
+    std_flag = get_cxx_std_flag(cc)
+    if std_flag is not None:
+        args.append(std_flag)
+
+    if sys.platform == 'darwin':
+        # Set min macOS version
+        min_macos_flag = '-mmacosx-version-min=10.9'
```

Comment:
I thought this would be consistent with `MACOSX_DEPLOYMENT_TARGET` that's used in the `scipy-wheels` repo, however what I see there with a quick search is still `10.6`. Which is weird, because I'm sure we dropped <10.9 compatibility.

Either way, this flag and the env var may clash, and I'm not sure which one takes precedence. Worth testing perhaps - would it make sense to only define this flag if `MACOSX_DEPLOYMENT_TARGET` is not defined?

Reply:
We do strictly require a minimum of 10.9 in order to build `uarray` and 10.9 is used in the wheel names. So, I think the `scipy-wheels` should be updated here.

The compiler flag will take precedence over the environment setting: 
https://stackoverflow.com/a/25362535/2556706

Reply:
Can you open a wheels repo issue for me to take a look at?

Reply:
Actually, on scipy-wheels I see 10.9 already:
https://github.com/MacPython/scipy-wheels/blob/193aaf07aa0d61805c317d8a40e7009b267d9876/env_vars.sh#L2

Reply:
thanks, just wanted to double check--I am a little behind on wheels work

Reply:
Thanks for checking. I found remnants in a few other places, will clean them up.